windows/wintools_uninstall_verify/add_new_devices_check_status.yml

```diff
@@ -40,6 +22,19 @@
     - new_portgroup is defined
     - new_portgroup.changed
 
+# Add a new pvscsi controller to VM
+- include_tasks: ../../common/vm_hot_add_remove_disk_ctrl.yml
+  vars:
+    disk_controller_ops: "present"
+    disk_controller_type: "paravirtual"
+- name: "Check add new pvscsi controller config changes"
+  ansible.builtin.assert:
+    that:
+      - disk_controller_facts is defined
+      - disk_controller_facts.changed
+    fail_msg: "Add new pvscsi controller task result 'changed' is not true."
```

Comment:
Can this assert task be moved into `common/vm_hot_add_remove_disk_ctrl.yml`, so that every time device change can be checked.

Reply:
Since there is no context in the common task, so we don't know if the task should be changed or not due to if there is existing controller, then the result is not changed.

Reply:
Got it. Thanks~